API.md

```diff
@@ -107,13 +107,17 @@ server.views({
         }
       },
       prepare: (options, next) => {
-        options.compileOptions.environment = nunjucks.configure(['node_modules/govuk-frontend', ...options.path])
+        options.compileOptions.environment = nunjucks.configure([
+          'node_modules/govuk-frontend',
+          'node_modules/@envage/hapi-govuk-question-page',
```

Comment:
This isn't necessary here as it's passed in on the path - I know this is just some example code so it doesn't really matter but I think having incorrect examples is misleading.